tests/test_live.py

```diff
@@ -0,0 +1,265 @@
+import io
```

Comment:
Kudos on being thorough with the tests.

I think its best to simplify these tests though. By that I mean get the coverage of live by executing as little other code as possible. Which probably means using a different renderable other than a table (which is the most complex renderable). A Text instance is probably enough.

I'd also prefer to avoid the text files and have the expected output embedded.

BTW there is a slight lower level way of capturing output, that I find works better for tests:

```python
console.begin_capture()
console.print("Hello")
assert console.end_capture() == "Hello\n"
```